tests/func/test_django.py

```diff
@@ -105,7 +105,7 @@ def test_django_breakpoint_no_multiproc(bp_target, start_method):
 
 
 @pytest.mark.parametrize('start_method', ['launch', 'attach_socket_cmdline'])
-@pytest.mark.skip(reason='Bug #694')
+@pytest.mark.skipif(sys.version_info < (3, 0), reason='Bug #923')
```

Comment:
#923 is closed no repro - should it be reopened?

Reply:
This doesn't seem right to me... that exception `AttributeError: 'thread' has no attribute 'frame'` means a real problem (it means that the debugger is stopping in a file inside the debugger -- this shouldn't happen because we should not trace those files at all).

Note that there was a recent fix for this where `ptvsd` files could end up being traced (https://github.com/Microsoft/ptvsd/commit/fd2aea41f3619b2a4e263f3151db39b1a5d29277), but if that's not the case and you have a repro, please reopen the issue and assign it to me as I think this would be a critical bug.

Reply:
I wanted to turn it off on 2.7, missed updating the reason. Since it is now closed, i removed that from some of the other tests as well.